packages/flutter_tools/lib/src/migrations/swift_package_manager_integration_migration.dart

```diff
@@ -540,15 +548,15 @@ $newContent
     return lines;
   }
 
-  bool _isNativeTargetMigrated(
+  bool _areNativeApplicationTargetsMigrated(
     ParsedProjectInfo projectInfo, {
     bool logErrorIfNotMigrated = false,
   }) {
     final bool migrated =
         projectInfo.nativeTargets
             .where(
               (ParsedNativeTarget target) =>
-                  target.identifier == _runnerNativeTargetIdentifier &&
```

Comment:
Instead of relying on the fixed identifier that only applies for the target that is created using the project template,
I updated this to work for _all_ application targets.